app/3.0/docs/stackblitz.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Polymer 3.0 on Stackblitz"
+---
+
+<!-- toc -->
+
+[StackBlitz](https://stackblitz.com) is an online editor for web projects. You can use StackBlitz to prototype and preview Polymer elements.
+
+Here's a [template for Polymer 3.0 to help you get started](https://stackblitz.com/edit/start-polymer3?file=start-polymer3.js). 
```

Comment:
Quick note: having this link means that you'll have to eventually update the project to have 3.0.0 as a dependency instead of the prerelease

Reply:
Also in polymer pre.13 we will need to change webcomponents-lite to webcomponents-bundle and webcomponentsjs to version 2.0.0-0

Reply:
https://github.com/Polymer/docs/issues/2551